config/hexa_control_odomless.yaml

```diff
@@ -0,0 +1,8 @@
+# Serial
+SerialPort: /dev/ttyACM0
+
+# Baudrate: 1 - 1000000, 16 - 115200, 34 - 57600
+SerialBaudrate: 1
+
+# Odometry Topic
+OdomEnable: false
```

Comment:
I vote we enable odometry by default, as I can imagine myself searching for hours why the odometry is not activated / learning does not happen as expected.